ui/raidboss/data/06-ew/raid/p12s.txt

```diff
@@ -4,3 +4,169 @@ hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+# P1 Athena
+0.0 "--sync--" sync /Engage!/ window 0,1
+1.0 "--sync--" sync / 03:[^:]*:Hemitheos:00:5A:0{4}:00::12383:/ window 10,3 jump 1000.5 # Sync to P2 immediately through AddCombatant.
+11.1 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/ window 20,20
+21.2 "Paradeigma" sync / 1[56]:[^:]*:Athena:82ED:/
+24.3 "--sync--" sync / 1[56]:[^:]*:Athena:8315:/
+36.7 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+37.5 "White Flame" sync / 1[56]:[^:]*:Anthropos:82F0:/
+39.3 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+41.9 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
```

Comment:
```suggestion
36.7 "Trinity of Souls 1" sync / 1[56]:[^:]*:Athena:(82E7|82E8):/
37.5 "White Flame" sync / 1[56]:[^:]*:Anthropos:82F0:/
39.3 "Trinity of Souls 2" sync / 1[56]:[^:]*:Athena:(82E9|82EA):/
41.9 "Trinity of Souls 3" sync / 1[56]:[^:]*:Athena:(82EB|82EC):/
```

I looked at the wing abilities too much, so I know there's only a limited amount of options here.

Reply:
Good to know! I wasn't having any issues with it in-game, but anything to keep it more efficient. 82E[78] and the like would probably be better, but I guess if we start going that far it'd be best to start using regex searches that use carets...